DoctrineMigrationsBundle.php

```diff
@@ -14,6 +14,7 @@
 
 namespace Doctrine\Bundle\MigrationsBundle;
 
+use Symfony\Component\Console\Application;
```

Comment:
minor, but this addition isn't needed now

Reply:
fixed in 5137def